server/repository/src/migrations/v1_05_00/cold_chain.rs

```diff
@@ -0,0 +1,13 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    #[cfg(feature = "postgres")]
+    sql!(
```

Comment:
can do cfg! macro to avoid warning

Reply:
hm? like this?
```
if cfg!(feature = "postgres") {
        sql!(
            connection,
            r#"
        ALTER TYPE permission_type ADD VALUE IF NOT EXISTS 'COLD_CHAIN_API';
        "#,
        )?;
    }
```

Reply:
yep, i am a fan of not using feature flags is possible (to check compilation for everything)